Tasks/VsTestV2/task.json

```diff
@@ -462,27 +462,32 @@
             "visibleRule": "rerunFailedTests = true"
         }
     ],
-    "sourceDefinitions": [
+    "dataSourceBindings": [
         {
             "target": "testPlan",
-            "endpoint": "/$(system.teamProject)/_apis/test/plans?filterActivePlans=true&api-version=3.0-preview.2",
-            "selector": "jsonpath:$.value[*].name",
-            "keySelector": "jsonpath:$.value[*].id",
-            "authKey": "tfs:teamfoundation"
+            "endpointId": "tfs:teamfoundation",
+            "endpointUrl": "{{endpoint.url}}/{{system.teamProject}}/_apis/test/plans?filterActivePlans=true&api-version=3.0-preview.2&$skip={{skip}}&$top=1000",
+            "resultSelector": "jsonpath:$.value[*]",
+            "resultTemplate": "{ \"Value\" : \"{{{id}}}\", \"DisplayValue\" : \"{{{id}}} - {{{name}}}\" }",
+            "callbackContextTemplate": "{\"skip\": \"{{add skip 1000}}\"}",
+            "callbackRequiredTemplate": "{{isEqualNumber result.count 1000}}",
+            "initialContextTemplate": "{\"skip\": \"0\"}"
         },
         {
             "target": "testConfiguration",
             "endpointId": "tfs:teamfoundation",
-            "endpoint": "/$(system.teamProject)/_apis/test/configurations?api-version=3.0-preview.1",
-            "selector": "jsonpath:$.value[*].name",
-            "keySelector": "jsonpath:$.value[*].id",
-            "authKey": "tfs:teamfoundation"
+            "endpointUrl": "{{endpoint.url}}/{{system.teamProject}}/_apis/test/configurations?api-version=3.0-preview.1",
+            "resultSelector": "jsonpath:$.value[*]",
+            "resultTemplate": "{ \"Value\" : \"{{{id}}}\", \"DisplayValue\" : \"{{{id}}} - {{{name}}}\" }"
         },
         {
             "target": "testSuite",
-            "endpoint": "/$(system.teamProject)/_apis/test/plans/$(testPlan)/suites?$asTreeView=true&api-version=3.0-preview.2",
-            "selector": "jsonpath:$.value[*]", 
-            "authKey": "tfs:teamfoundation"
+            "endpointId": "tfs:teamfoundation",
+            "endpointUrl": "{{endpoint.url}}/{{system.teamProject}}/_apis/test/plans/{{testPlan}}/suites?$asTreeView=true&api-version=3.0-preview.2",
```

Comment:
I don't see pagination in test suite. is this expected?

Reply:
Yes.